XenModel/Messages.resx

```diff
@@ -12061,7 +12061,7 @@ Check your settings and try again.</value>
     <value>No Default SR on Pool '{0}'</value>
   </data>
   <data name="UPDATES_WIZARD_NO_HOSTS" xml:space="preserve">
-    <value>There are no servers available to migrate the VM {0} to.</value>
+    <value>There are no servers available to migrate the VM '{0}' on '{1}' to.</value>

```

Comment:
I think it would be better to have the host name at the begining, as we have in other problems.
_{1}: There are no servers available to migrate the VM {0} to._

Reply:
Agree.
I've aligned my code with "other problems".